plugins/source/aws/docs/tables/aws_ec2_managed_prefix_lists.md

```diff
@@ -0,0 +1,28 @@
+# Table: aws_ec2_managed_prefix_lists
+
+https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_ManagedPrefixList.html. 
+The 'request_account_id' and 'request_region' columns are added to show the account_id and region of where the request was made from.
+
+The composite primary key for this table is (**request_account_id**, **request_region**).
+
+## Columns
+
+| Name          | Type          |
+| ------------- | ------------- |
+|_cq_source_name|String|
+|_cq_sync_time|Timestamp|
+|_cq_id|UUID|
+|_cq_parent_id|UUID|
+|request_account_id (PK)|String|
```

Comment:
What's the reason for the `request_` prefix? This table doesn't have any other `region` or `account columns

Reply:
The `OwnerID` is actually the `account_id` that owns the prefix list... So didn't want users to get confused about the `account_id` field... And kept the `request_region` in there just to be consistent...